File: WebAPI/WebAPI/Models/Seat.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json.Serialization;
using System.Threading.Tasks;

namespace WebAPI.Models
{
	public class Seat
	{
		public int SeatId { get; set; }
		public bool Reserved { get; set; }
		public string Name { get; set; }
		public int FlightId { get; set; }
		[JsonIgnore]
		public Flight Flight { get; set; }

		public Seat()
		{

		}
	}
}

File: Angular9/src/entities/seat.ts
import { Stop } from './stop';
import { Flight } from './flight';

export class Seat
{
   seatId: number;
   reserved: boolean;
   name: string;
   flightId: number;
   flight: Flight;

    // constructor(dtascend: string, dtdescend: string, duration: number, distance: number, ticketprice: number, stops: Array<Stop>)
    // {
    //     this.distance = distance;
    //     this.dtascend = dtascend;
    //     this.dtdescend = dtdescend;
    //     this.ticketprice = ticketprice;
    //     this.stops = stops;
    //     this.duration = duration;
    // }
    /**
     *
     */
    constructor(name: string) {
        this.reserved = false;
        this.name = name
    }
}